src/main/java/com/github/ermay12/regex/Regex.java

```diff
@@ -669,13 +702,14 @@ public static Regex concatenate(Regex... components) {
   /**
    * Returns a regex which matches one of the given strings. If no strings
    * are provided in the arguments, a regular expression matching nothing is returned
+   *
    * @param ss the CharSequences that are the options
    * @return a regex which matches one of the given regular expressions
    */
   public static Regex oneOf(CharSequence... ss) {
-    assert(ss.length > 0); // Should not fail because you cannot call
-                           // oneOf with no arguments due to overloading
-    if(ss.length > 1) {
+    assert (ss.length > 0); // Should not fail because you cannot call
+    // oneOf with no arguments due to overloading
```

Comment:
what why did you delete this now the formatting is bad

Reply:
whoops auto-linter

Reply:
Can you un-auto-lint